install.py

```diff
@@ -226,20 +222,18 @@ def validate_path(path):
     cuda_dir = validate_path(cuda_dir)
     nccl_dir = validate_path(nccl_dir)
     tblis_dir = validate_path(tblis_dir)
-    legate_dir = validate_path(legate_dir)
     thrust_dir = validate_path(thrust_dir)
     curand_dir = validate_path(curand_dir)
     gasnet_dir = validate_path(gasnet_dir)
     cutensor_dir = validate_path(cutensor_dir)
     openblas_dir = validate_path(openblas_dir)
 
-    if legate_dir is None:
-        try:
-            import legate.install_info as lg_install_info
+    try:
+        import legate.install_info as lg_install_info
 
-            legate_dir = dirname(lg_install_info.libpath)
-        except Exception:
-            pass
+        legate_dir = dirname(lg_install_info.libpath)
+    except Exception:
+        legate_dir = None
```

Comment:
My only question is should this be an error instead?

Reply:
It looks like cmake exposes the capability to build legate.core as part of building cunumeric (see options `legate_branch` and `legate_url`), so I assume leaving `legate_dir` undefined (and failing to find `legate.install_info`) was meant to trigger this behavior?

However, I recall reading that (at least currently) a recursive build of legate.core through cunumeric will not do a complete enough installation (e.g. `legion_python` and the legate driver will not get installed). Therefore it might be a good idea to require that `legate.install_info` is found at this stage (and correspondingly remove `legate_url` and `legate_branch`)?

@trxcllnt do I have it right?